lib/engine/game/base.rb

```diff
@@ -79,6 +79,14 @@ def self.title
         name.split('::').last.slice(1..-1)
       end
 
+      def self.register_colors(colors)
+        colors.default_proc = proc do |_, key|
+          key
+        end
```

Comment:
This makes sure that when you don't define a certain color it just passes the color name through (which works fine in SVG)